README.md

```diff
@@ -38,6 +38,23 @@ $prefix/data/
 
 `RELATIVE` is always set when building for Windows or macOS.
 
+## Cross-compiling on Linux for Windows
+
+Get MinGW equivalents of listed dependencies, then run:
+
+```
+mkdir build
+cd build
+mkdir /tmp/taisei-build
+x86_64-w64-mingw32-cmake -DCMAKE_SYSTEM_NAME=Windows -DCMAKE_C_COMPILER=/usr/bin/x86_64-w64-mingw32-gcc -DCMAKE_INSTALL_PREFIX=$output-folder -DRELATIVE=TRUE -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
The right thing to do here is to use a toolchain file, like the one generated here:
https://github.com/lachs0r/mingw-w64-cmake/blob/master/toolchain.cmake.in